src/test/shell/bazel/bazel_sandboxing_test.sh

```diff
@@ -309,6 +317,68 @@ EOF
   bazel build //pkg:a &>$TEST_log || fail "expected build to succeed"
 }
 
+# Sets up targets under //test that, when building //test:all, verify that the
```

Comment:
I introduced this check instead of checking for whether `/tmp` is shared with the host directly. We don't care so much about the latter, so I felt that encoding it in tests would make them more brittle as we explore alternative options.